src/components/PostsList.tsx

```diff
@@ -1,86 +1,74 @@
 import React from 'react';
+import { Post } from '../types/Post';
+import classNames from 'classnames';
 
-export const PostsList: React.FC = () => (
-  <div data-cy="PostsList">
-    <p className="title">Posts:</p>
+type Props = {
+  userPosts: Post[];
+  selectedPost: Post | null;
+  setShowForm: (value: boolean) => void;
+  setSelectedPost: (postId: Post | null) => void;
+};
 
-    <table className="table is-fullwidth is-striped is-hoverable is-narrow">
-      <thead>
-        <tr className="has-background-link-light">
-          <th>#</th>
-          <th>Title</th>
-          {/* eslint-disable-next-line jsx-a11y/control-has-associated-label */}
-          <th> </th>
-        </tr>
-      </thead>
+export const PostsList: React.FC<Props> = ({
+  userPosts,
+  selectedPost,
+  setShowForm,
+  setSelectedPost = () => {},
```

Comment:
Setting a default value for `setSelectedPost` to an empty function can lead to unexpected behavior if the prop is not provided. Consider making this prop required without a default value.